packages/core/src/QueryManager.js

```diff
@@ -18,10 +18,24 @@ export class QueryManager {
   }
 
   next() {
+    /*
     if (this.pending || this.queue.isEmpty()) return;
     const { request, result } = this.queue.next();
     this.pending = this.submit(request, result);
     this.pending.finally(() => { this.pending = null; this.next(); });
+    */
+    /* */
```

Comment:
Please don't comment out code in a pull request but remove it so we can see the diff.